backend/app/Services/GoogleService.php

```
@@ -264,24 +269,61 @@ public function createEvent(
     }
 
+    /**
+     * Patch the end time of an existing Google Calendar event.
+     */
+    public function patchEventEndTime(
+        GoogleAccount $googleAccount,
+        Calendar $calendar,
+        string $eventId,
+        Carbon $newEnd
+    ): void {
+        $bookingMethod = $googleAccount->booking_method ?? GoogleBookingMethod::USER_ACCOUNT;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$bookingMethod'. (undefined)

(UnusedLocalVariable)